src/test/unit_tests/test_system_servers.js

```diff
@@ -435,6 +436,36 @@ mocha.describe('system_servers', function() {
         });
     });
 
+    mocha.it('Calculate md5_etag for bucket works', async function() {
+        this.timeout(90000); // eslint-disable-line no-invalid-this
+        return P.resolve()
+            .then(() => rpc_client.bucket.create_bucket({
+                name: MD5_BUCKET,
+                force_md5_etag: true,
+            }))
+            .then(() => rpc_client.bucket.read_bucket({
+                name: MD5_BUCKET,
+            }))
+            .then(info => assert(info.force_md5_etag && info.force_md5_etag === true))
+            .then(() => rpc_client.bucket.update_bucket({
+                name: MD5_BUCKET,
+                force_md5_etag: false,
+            }))
+            .then(() => rpc_client.bucket.read_bucket({
+                name: MD5_BUCKET,
+            }))
+            .then(info => assert(info.force_md5_etag && info.force_md5_etag === false))
```

Comment:
this condition will always be false. suggest changing to this


```suggestion
            .then(info => assert(info.force_md5_etag === false))
```

Reply:
Okay, I will do the required changes here.